lib/util/bundletool.js

```diff
@@ -0,0 +1,202 @@
+var cp = require('child_process')
+var fs = require('fs')
+var path = require('path')
+var request = require('request')
+
+var Promise = require('bluebird')
+var yauzl = require('yauzl')
+
+var logger = require('./logger')
+
+module.exports = function(options) {
+  return new Promise(function(resolve, reject) {
+    var defaultBaseDir = ('/tmp/bundletool')
+    var bundletoolUrl = 'https://api.github.com/repos/google/bundletool/releases/latest'
```

Comment:
IMO bundletool should be bundled with docker image.
STF may be deployed in various environments which may not have network access.